cannon/mipsevm/memory/memory.go

```diff
@@ -193,8 +194,8 @@ func (m *Memory) pageLookup(pageIndex Word) (*CachedPage, bool) {
 }
 
 func (m *Memory) SetUint32(addr Word, v uint32) {
-	// addr must be aligned to WordSizeBytes bytes
-	if addr&arch.ExtMask != 0 {
+	// addr must be aligned to 4-byte (32-bit) boundaries
+	if addr&3 != 0 {
```

Comment:
Calling out this unrelated change here - I think we want to be able to set uint32's at any even multiple of 4 bytes.

Reply:
The only usecase I can think of to do this is for writing 4-byte instructions at 4-byte boundaries. I'd prefer that those callsites should be modified to use 8-byte aligned addresses.
Is any other usecase for this for the 64-bit VM? Relaxing the alignment constraint here breaks my mental model of how stores are emulated for a 64-bit CPU.

Reply:
I think this method is exclusively used in tests.  So we should probably just cut this from the production code altogether, and make a test util or else migrate the existing calls to use 8-byte boundaries as you suggest.  I can revert my change for now and add a ticket to clean this up.

Reply:
+1 on removing this from production code. We have delegate it to a testutil helper function that only sets up the instruction word.

Reply:
Added a ticket [here](https://github.com/ethereum-optimism/optimism/issues/12488).  Should I add another ticket to cut `memory.GetUint32`?

Reply:
Thanks for creating the ticket. That issue can tackle both.